react-effects/my-app/src/List.jsx

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-unused-vars -- Remove me */
+import { useEffect, useState } from 'react';
+import readItems from './read';
+
+export default function List() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [items, setItems] = useState([]);
+  const [error, setError] = useState();
+
+  // Your code here:
+  //  - When the component mounts:
+  //    - Read the items using `readItems` and update state so the list displays
+  //    - Handle errors from `readItems`
+
+  useEffect(() => {
+    async function getItems() {
+      try {
+        const items = await readItems();
+        setIsLoading(false);
+        setItems(items);
+      } catch (err) {
+        setIsLoading(false);
```

Comment:
Nice.
The easiest way to set isLoading to false is to do it in the `finally` clause.